tbx/project_styleguide/templates/patterns/pages/work/work_detail.html

```diff
@@ -17,29 +17,75 @@
             'event': 'page-information'
         });
     </script>
-    <meta name="author" content="{{ page.authors.all.0.author.name }}">
+    <meta name="author" content="{{ page.first_author.name }}">
     <meta property="og:article:published_time" content="{{ page.date|date:'Y-m-d' }}" />
     {{ block.super }}
 
 {% endblock %}
 
 {% block content %}
-    {% pageurl page.work_index as work_index_url %}
-    {% with author=page.authors.first.author %}
+    <div class="grid grid--spacer-large streamfield">
+        <div class="authors">
+            {% with author=page.first_author %}
+                {% include "patterns/molecules/author/author.html" with author=author %}
+            {% endwith %}
+        </div>
+        <div class="streamfield__title">
+            <h1 class="heading heading--one-b">{{ page.title }}</h1>
 
-        {% include "patterns/molecules/title-filters/title-filters.html" with item=page %}
-        {% include "patterns/molecules/author/author.html" with item=page author=author tag_link_base=work_index_url %}
+            {% pageurl page.work_index as work_index_url %}
 
-        {% include_block page.body %}
+            <div class="tags tags--with-title">
+                <span class="sr-only">Related work categories</span>
+                {% for tag in page.related_services.all %}
+                    {% include "patterns/atoms/tag/tag.html" with tag_slug=tag.slug tag_name=tag.name tag_link_base=work_index_url %}
+                {% endfor %}
+            </div>
+        </div>
 
-        {% include "patterns/molecules/author/author.html" with item=page author=author tag_link_base=work_index_url %}
+        {% for intro_block in page.body %}
+            {% if intro_block.block_type == 'intro' %}
+                {% include_block intro_block %}
+            {% endif %}
+        {% endfor %}
 
-        {% include_block page.call_to_action %}
+        <div class="work-meta mt-spacerSmall mb-spacerSmall lg:mb-spacerLarge">
+            {% if page.client %}
+                <p class="work-meta__client">
+                    {{ page.client }}
+                </p>
+            {% endif %}
+
+            <p class="work-meta__meta">
+                <time datetime="{{ item.date|date:'Y-m-d' }}">{{ page.date }}</time>
```

Comment:
I wonder if we should set the date format globally to `|date:'Y-m-d'` so we don't have to add the filter everywhere it's used.

Reply:
good idea I'll check how to do that

Reply:
I'll add this to a list as for now I'm not sure and I'd like to get this merged.